streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MenuServiceImpl.java

```diff
@@ -133,7 +137,9 @@ public void updateMenu(Menu menu) throws Exception {
     public void deleteMenus(String[] menuIds) throws Exception {
         for (String menuId : menuIds) {
             // Find users associated with these menus/buttons
-            this.baseMapper.deleteRoleMenuByMenuId(Long.parseLong(menuId));
+            LambdaQueryWrapper<RoleMenu> queryWrapper = new LambdaQueryWrapper<>();
+            queryWrapper.eq(RoleMenu::getMenuId, Long.parseLong(menuId));
+            this.roleMenuMapper.delete(queryWrapper);
             // Recursively delete these menus/buttons
             this.baseMapper.deleteById(menuId);
```

Comment:
I recommend to use this method: `this.removeById(menuId);`

Reply:
Done